media-sound/guayadeque/guayadeque-0.4.3.ebuild

```diff
@@ -0,0 +1,106 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+WX_GTK_VER=3.0
+
+inherit cmake-utils wxwidgets
+
+DESCRIPTION="Music management program designed for all music enthusiasts"
+HOMEPAGE="http://guayadeque.org/"
+SRC_URI="https://github.com/anonbeat/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="ayatana ipod +minimal"
+
+# No test available, Making src_test fail
+RESTRICT="test"
+
+GST_VERSION=1.0
+GST_DEPS="
+	dev-perl/GStreamer
+	media-libs/gnonlin:${GST_VERSION}
+	media-plugins/gst-plugins-libav:${GST_VERSION}
+	media-plugins/gst-plugins-libnice:${GST_VERSION}
+	media-plugins/gst-plugins-pulse:${GST_VERSION}
+	media-plugins/gst-plugins-soup:${GST_VERSION}
+	media-libs/gst-plugins-bad:${GST_VERSION}
+	media-libs/gst-plugins-base:${GST_VERSION}
+	media-libs/gst-plugins-good:${GST_VERSION}
+	media-libs/gst-plugins-ugly:${GST_VERSION}
+"
```

Comment:
I was reviewing this whole list, but firefox or github ate my big comment. So here's more to do, but I need to re-do this checking and draft it elsewhere, so just a note for now to post the rest of the review for now